README.md

```diff
@@ -1,26 +1,45 @@
 ## Description
-These rules refer to 
+These rules refer to http://developer.android.com/guide/developing/device.html and include many suggestions from the Archlinux and Github Community.
 
-http://developer.android.com/guide/developing/device.html
```

Comment:
On most linux instances that I've  run into (Fedora guy mostly myself)  there are no default  android udev rules   (would recommend dropping the 'Ubuntu' part tho but the rest of the statement still rings true.